airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/decd338e-5647-4c0b-adf4-da0e75f5a750.json

```diff
@@ -2,6 +2,6 @@
   "sourceDefinitionId": "decd338e-5647-4c0b-adf4-da0e75f5a750",
   "name": "Postgres",
   "dockerRepository": "airbyte/source-postgres",
-  "dockerImageTag": "0.2.1",
+  "dockerImageTag": "0.2.2",
```

Comment:
I think there's something we don't handle very well at the moment is conflict in connector's versions being bumped in multiple branches at the same time... I'm not sure how we should resolve this though.

So when you'll try to bump versions of those connectors, it's going to fail because 0.2.2 has already been published but not merged yet...:

https://hub.docker.com/r/airbyte/source-postgres/tags?page=1&ordering=name

Reply:
that makes sense. will coordinate with artem!